### ISSUE TITLE
Periodic Reports | Skipped Boolean & Reason

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-api-v3#1998]

**Comment by CarsonF** [comment]
@sethmcknight Should reports that are skipped be ignored when looking for next & current reports due?

**Comment by sethmcknight** [comment]
@CarsonF no, we'll want to take them into account still. For example if we say the 'next due' report is being skipped for xyzzy reason, we still want to show that is the next report due (until it's the current report) so that we can see it was skipped vs just not submitted.

**Comment by CarsonF** [comment]
For now let's add an "Skip" option to this context menu. It can bring up a dialog modal with a textarea to describe why it was skipped.
![Screen Shot 2021-09-28 at 5 13 37 PM](https://user-images.githubusercontent.com/932566/135173173-03e9b671-44ba-419a-8d99-83eef5dc58d2.png)

In the card, we can say "Skipped" instead of "due ...". Maybe the reason can be displayed in a `<PaperTooltip>`
![Screen Shot 2021-09-28 at 5 16 05 PM](https://user-images.githubusercontent.com/932566/135173296-04bf417e-8406-439c-87b5-8a27704f22a8.png)
